_includes/panels/panel_action_w_call_congress.html

```diff
@@ -9,10 +9,10 @@
       <p>Join the Mayday team of Citizen Lobbyists fighting together for fundamental reform.</p>
       <form class="join" id="js-form-call-congress">
         <div class="row">
-          <div class="col-md-8">
+          <div class="col-md-8 form-left-sm">
```

Comment:
I had adding new classes if we can avoid them.  I'm guessing you explored that, but just try to keep minimizing where possible.  Thanks!